lighthouserc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
Seems like it would be helpful to add some comments here. Might be helpful to link to the documentation for the structure of this file and configuration options.